persistence/sql/persister_identity.go

```diff
@@ -281,6 +282,43 @@ func (p *Persister) ListIdentities(ctx context.Context, page, perPage int) ([]id
 	return is, nil
 }
 
+func (p *Persister) ListIdentitiesFiltered(ctx context.Context, values url.Values, page, perPage int) ([]identity.Identity, error) {
+	is := make([]identity.Identity, 0)
```

Comment:
Guess you can add a capacity here already.
```suggestion
	is := make([]identity.Identity, 0, perPage)
```

Reply:
good point